### ISSUE TITLE
The synthesis images are not smoothness

### ISSUE THREAD

**Issue by qinhuafeng** [costapt/vess2ret#18]
Dear Pedro Costa，
    I am very interesting to  End-to-End Adversarial model for retinal images synthesis. It is an very awesome approach to generate retinal image. Therefore，our group has reproduced your results based on your code available at Github and employed your approach for vein image synthesis. In our experimental results， we found that  the generated images from training set is perfect， but these from validation set and test set are not smoothness and shown borders of the patches clearly.   Also， we test your model based on your demo. When we submit retinal images shown in your demo，the synthesis images are smoothness. However，when we submit retinal images which do not show in your demo，the generated images shown borders of the patches clearly. We try to increase value of the alpha parameter，but it still does not solve the problem. Do you have similar problem and how to solve it？We would appreciate it that you can provide help for us.  Thank you very much！
Regards
Huafeng Qin

**Comment by qinhuafeng** [comment]
[](url)

**Comment by little1TouwaErio** [comment]
@ @qinhuafeng Hello, may I see the effect of your training test with your own dataset？Because I am not sure whether this network is helpful to my dataset, I hope to get your reply as soon as possible.
